pkg/controllers/status/cluster_status_controller.go

```diff
@@ -153,7 +153,11 @@ func (c *ClusterStatusController) Reconcile(ctx context.Context, req controllerr
 		return controllerruntime.Result{Requeue: true}, nil
 	}
 
-	return c.syncClusterStatus(ctx, cluster)
+	err := c.syncClusterStatus(ctx, cluster)
+	if err != nil {
+		return controllerruntime.Result{}, err
+	}
+	return controllerruntime.Result{RequeueAfter: c.ClusterStatusUpdateFrequency.Duration}, nil
```

Comment:
Is this a bug that the status controller didn't report the status as per `ClusterStatusUpdateFrequency.Duration`?

Reply:
No,  i see. ignore me.